KnowSheet/scripts/full-test.sh

```diff
@@ -11,34 +11,37 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -fprofile-arcs -ftest-coverage -DBRICKS_COVERAGE_REPORT_MODE"
 LDFLAGS="-pthread"
 
-TMPDIR=.noshit/fulltest
-TMPDIR_FULL_PATH=$(readlink -e .)/.noshit/fulltest
+SCRIPT_DIR=$( $( dirname "${BASH_SOURCE[0]}" )/KnowSheetReadlinkDir.sh "${BASH_SOURCE[0]}" )
+RUN_DIR="$PWD"
 
-GOLDEN_SUBDIR_NAME=golden
-GOLDEN_FULL_PATH=$TMPDIR_FULL_PATH/$GOLDEN_SUBDIR_NAME
+TMPDIR=".noshit/fulltest"
+TMPDIR_FULL_PATH="$SCRIPT_DIR/.noshit/fulltest"
```

Comment:
Same here: Relative to the current directory, not the one the script is located at.